model/src/test/kotlin/RootLicenseMatcherTest.kt

```diff
@@ -118,7 +118,7 @@ private fun licenseFindings(vararg paths: String): List<LicenseFinding> =
     paths.map {
         LicenseFinding(
             license = "LicenseRef-not-relevant",
-            location = TextLocation(it, startLine = 1, endLine = 2)
+            location = TextLocation(it, 1, 2)
```

Comment:
I liked it with the named arguments, it removes the requirement to know or look up what the arguments mean. The style guide says:
"Use the named argument syntax when a method takes multiple parameters of the same primitive type, or for parameters of Boolean type, unless the meaning of all parameters is absolutely clear from context."
I'm not convinced the meaning here is absolutely clear from context.
I'm not saying we need to align this now for all places where we create `TextLocation`s, but I also don't see a need to remove the names where we have used them before.

Reply:
> it removes the requirement to know or look up what the arguments mean.

I tend to agree that makes sense for Booleans, but for Ints and in this specific case I found it to be rather obvious what they are about, esp. as the IDE shows you:

![image](https://user-images.githubusercontent.com/349154/101605533-917a7780-3a02-11eb-89c0-ddb26c9191d0.png)

> I also don't see a need to remove the names where we have used them before.

To me, the inconsistency of sometimes using named parameters for `TextLocation` and sometimes not *in the same file* was more bothering than not having named parameters at all.

Reply:
> > it removes the requirement to know or look up what the arguments mean.
> 
> I tend to agree that makes sense for Booleans, but for Ints and in this specific case I found it to be rather obvious what they are about, esp. as the IDE shows you:

Ok, but this does not help with code review. Btw, I thought that might be a reason to try code review in IntelliJ, but the PR diffs there also don't show the argument names. But at least you can easily access function docs there.

> > I also don't see a need to remove the names where we have used them before.
> 
> To me, the inconsistency of sometimes using named parameters for `TextLocation` and sometimes not _in the same file_ was more bothering than not having named parameters at all.

I agree for files where it is mixed, but the file where I put my comment has only one call.

Reply:
So, how do we go forward here? Do you want me to restore named parameters in those files where *all* `TextLocations` already used named parameters?

Reply:
I would have preferred to keep the named arguments and rather align it in the other direction, but I don't care too much.